README.md

```diff
@@ -0,0 +1,38 @@
+# MBTA Schemas
+
+A collection of schemas for inter-app interchange formats. Schemas in this repository should stay up to date if they change.
+
+Schemas go in the `schemas/` folder.
+
+Examples go in the `examples/<schema>/` folders.
```

Comment:
What if the schema was in `<schema>/schema.json` and the examples were the other files in `<schema>/`? I think storing the examples alongside the schema might be easier to maintain than having one folder for schemas and one folder for examples, but I'm not sure.

Reply:
Thinking through this...

I guess some use cases might want multiple schema files to reference each other, so having an app folder instead of a single `schemas/<schema>.json` file is good.

I'd probably want `schemas/<app>/<schema>.json` and `schemas/<app>/examples/`, so apps don't get mixed up with all the CI stuff that ended up top level, and so you can tell schema json files apart from example json files.

If there are multiple schemas in a folder, I'm not sure how the tests should know which schema to apply against which example. Perhaps `schemas/<app>/examples/<schema>/<whateverexample>.json`? matches to `schemas/<app>/<schema>.json`?

But that's all overengineered / YAGNI. The important part for now is that the current glides schema is tested, and the example folder isn't a problem at this scale, so I want to leave it as is. If a future use case decides that they need a more featured / more organized naming scheme, then they can do it then. It'd just be file moves, so wouldn't be a big change in git.